test/functional/testCases.js

```diff
@@ -513,6 +513,72 @@ const testCases = [
             }
         ]
     },
+    {
+        describeName: '0022   Simple group with active attributes and multimeasures',
+        provision: {
+            url: 'http://localhost:' + config.iota.server.port + '/iot/services',
+            method: 'POST',
+            json: {
+                services: [
+                    {
+                        resource: '/iot/json',
+                        apikey: globalEnv.apikey,
+                        entity_type: globalEnv.entity_type,
+                        commands: [],
+                        lazy: [],
+                        attributes: [
+                            {
+                                object_id: 'a',
+                                name: 'attr_a',
+                                type: 'Number'
+                            }
+                        ],
+                        static_attributes: []
+                    }
+                ]
+            },
+            headers: {
+                'fiware-service': globalEnv.service,
+                'fiware-servicepath': globalEnv.servicePath
+            }
+        },
+        should: [
+            {
+                loglevel: 'debug',
+                shouldName:
+                    'A - WHEN sending defined object_ids (measures) through http IT should send measures to Context Broker preserving value types and name mappings',
+                type: 'multimeasure', // TBD: this should be implemented to expect /v2/op/update
```

Comment:
Multimeasures were supported previously as described [here](https://github.com/telefonicaid/iotagent-node-lib/tree/master/test/functional#multimeasures)

> It is also supported to test cases in which is sent more than one measure. To do so, you need to define the test case expectation as an array, with one object for each measurement. Then, the suite will recognize the array length and will expect the same number of NGSI requests. I.E:

Reviewing the testsuite at this part of the code:
https://github.com/telefonicaid/iotagent-node-lib/blob/8b86b0f8885072b15072939760da4ff3c0a1478a/test/functional/testUtils.js#L169-L177

When defining the type as `multientity` seems it expect `/v2/op/update` which I uderstand is the requiremet

Reply:
Note that `multimeasure` is not included in the documentation at https://github.com/telefonicaid/iotagent-node-lib/tree/master/test/functional/README.md

> `type`: The type of the test case. This can be `single` or `multientity`. See the "Advanced features" section for more information.

If we have invented a new type in this PR, I understand it should be properly documented there.

Reply:
Interestingly, there is a section about multimeasures in such documentation https://github.com/telefonicaid/iotagent-node-lib/blob/master/test/functional/README.md#multimeasures, but the relation with the test `type` is not mentioned there...

Reply:
> Probably, something like this in the code should works:

function ```jsonToIotaMeasures(json)``  from testUtils.js should be updated to allow it.

Reply:
Done in https://github.com/telefonicaid/iotagent-node-lib/pull/1607/commits/94db695ed0dc7248cb4e4e1572f8069a07dad88c

But may be still could be refactored @mapedraza

Reply:
Done in https://github.com/telefonicaid/iotagent-node-lib/pull/1607/commits/94db695ed0dc7248cb4e4e1572f8069a07dad88c